Rules/CTF/Scripts/CTF.as

```diff
@@ -23,8 +23,11 @@ void Config(CTFCore@ this)
 
 	//how long to wait for everyone to spawn in?
 	s32 warmUpTimeSeconds = cfg.read_s32("warmup_time", 30);
-	this.warmUpTime = (getTicksASecond() * warmUpTimeSeconds);
+	this.warmUpTime = 1;//(getTicksASecond() * warmUpTimeSeconds);
```

Comment:
Please don't leave debug modifications in official pull requests 😅 will test this as-is though and wait for changes.